README.md

```diff
@@ -5,10 +5,8 @@ An HTTP client module for MicroPython with an API *similar* to [requests].
 This is an evolution of the [urequests] module from [micropython-lib] with a few
 extensions and many fixes and convenience features.
 
-
 ## Features & Limitations
```

Comment:
Please leave the empty lines before headings a they are. I like having two empty lines before section headers. So please revert those changes.

Reply:
These have somehow been automatically cleaned by VS Code.
For some reason `editor.formatOnSave` was enabled 🤦🏼‍♂️ 
Will be resolved in next commit